jcl/src/java.base/share/classes/java/lang/StackWalker.java

```diff
@@ -181,7 +182,11 @@ public void forEach(Consumer<? super StackFrame> action) {
 				s -> s.limit(2).collect(Collectors.toList()));
 		if (result.size() < 2) {
 			/*[MSG "K0640", "getCallerClass() called from method with no caller"]*/
+			/*[IF Sidecar19-SE-B165]
+			throw new IllegalCallerException(com.ibm.oti.util.Msg.getString("K0640")); //$NON-NLS-1$
+			/*[ELSE]*/
 			throw new IllegalStateException(com.ibm.oti.util.Msg.getString("K0640")); //$NON-NLS-1$
```

Comment:
Why do we need to retain this old behavior?

Reply:
Build 148 (i.e. vmfarm builds) doesn't have the new exception.